cmd/acr/purge.go

```diff
@@ -48,15 +49,18 @@ const (
   - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository, with 4 purge tasks running concurrently
 	acr purge -r example --filter "hello-world:.*" --ago 1d --concurrency 4
 	`
-	maxPoolSize              = 32 // The max number of parallel delete requests recommended by ACR server
-	manifestListContentType  = "application/vnd.docker.distribution.manifest.list.v2+json"
-	manifestIndexContentType = "application/vnd.oci.image.index.v1+json"
-	linkHeader               = "Link"
+	maxPoolSize               = 32 // The max number of parallel delete requests recommended by ACR server
+	MediaTypeManifestList     = "application/vnd.docker.distribution.manifest.list.v2+json"
+	MediaTypeArtifactManifest = "application/vnd.oci.artifact.manifest.v1+json"
+	MediaTypeImageManifest    = v1.MediaTypeImageManifest
+	MediaTypeImageIndex       = v1.MediaTypeImageIndex
```

Comment:
Is there a need to expose these media types?

Reply:
yes, I need to unmarshall the manifest byte if the manifest of interest is one of the media types from line 53 to line 56.

Reply:
I mean they don't need to be capitalized.